test/common/common/BUILD

```diff
@@ -114,6 +116,21 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_benchmark_binary(
+    name = "logger_perf_test",
+    srcs = ["logger_perf_test.cc"],
```

Comment:
We name this *_speed_test for rest benchmarks, can you rename this one too? same for below

Reply:
Addressed.